### ISSUE TITLE
Will there be any updates？

### ISSUE THREAD

**Issue by Ironmankiller** [mookixiao/simulatedsender#1]
I’m looking forward the new feature of this repository.

**Comment by mookixiao** [comment]
Yeah, I have added a feature that you can drag the slider to seek location in file where  to send next. 
Other features are on the way. 
If you have any feature you think it's necessary to add, feel free to leave comments.

**Comment by mookixiao** [comment]
> I’m looking forward the new feature of this repository.

Yeah, I have added a feature that you can drag the slider to seek location in file where  to send next. 
Other features are on the way. 
If you have any feature you think it's necessary to add, feel free to leave comments.
